sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.projectName=Horondi-Client-Fe
 sonar.projectVersion=1.0
 sonar.sources=.
 sonar.host.url=https://sonarcloud.io
-sonar.exclusions=public, src/**/*.test.js, src/**/**/*.test.js, src/**/**/**/*.test.js, src/**/**/**/**/*.test.js, src/**/**/**/**/**/*.test.js, cypress/**
+sonar.exclusions=public, src/**/*.test.js, src/**/**/*.test.js, src/**/**/**/*.test.js, src/**/**/**/**/*.test.js, src/**/**/**/**/**/*.test.js, cypress/**, src/locales/**/index.js
```

Comment:
Added this exclusion to not track locales config file with Sonar. Because these two files are actually the same for both languages.